client/src/app/project-form/project-form.component.ts

```diff
@@ -155,7 +155,7 @@ export class ProjectFormComponent implements OnInit {
           console.log('Retrieved project', projectResource);
           this.projectResource = projectResource;
           if (projectResource && projectResource.content &&
-            !projectResource.content.hasOwnProperty('describedBy') || !projectResource.content.hasOwnProperty('schema_type')) {
+            (!projectResource.content.hasOwnProperty('describedBy') || !projectResource.content.hasOwnProperty('schema_type'))) {
```

Comment:
This is fine but I think you can skip the first bit and do `projectResource?.content?.hasOwnProperty('describedBy')`

Reply:
Yes, thanks for that. I forgot the angular way. ;-)
I am going to refactor it like that.
We are also trying to add a test with @amnonkhen.
It is not straightforward to me, but hopefully we can do it.

Reply:
It's not anything angular, it's just new JS notation. Sounds like a good plan!

Reply:
Both are good practice.
Elvis ( ?. ) is always good, and adding unit tests is always good.
I feel we should have a workshop about Angular unit testing.